tests/lib/rules/space-infix-ops.js

```diff
@@ -358,6 +373,47 @@ ruleTester.run("space-infix-ops", rule, {
                 column: 6,
                 nodeType: "BinaryExpression"
             }]
+        },
+        {
+            code: "a+b - c",
+            options: { all: "never", "+": "always" },
+            output: "a + b-c",
+            errors: [{
```

Comment:
It makes sense to me why it should have two errors, but I don't believe it does. I didn't change that part of the code. I only changed the part that determines which operators should be processed by the existing code.